api/main_endpoints/models/password-reset.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
```

Comment:
this only change should be this single file, you can undo the other changes by resetting it with dev with `git checkout origin/dev filename`